### PR TITLE
Rename confusing fixtures, especially ones with the same name but dif…

### PR DESCRIPTION
**Pull request by aqk**
…ferent implementation

### CODE REVIEW

**Review by arvidn — DISMISSED** [review]
the submodule update looks suspicious. hopefully @seeden can confirm that this is what we want to do

**Comment by aqk** [comment]
@wjblanke Removed  chia-blockchain-gui changes.

See long discussion in IPS channel to understand M1 build test breakage

**Review by altendky — DISMISSED** [review]

File: tests/wallet/cat_wallet/test_cat_wallet.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ async def tx_in_pool(mempool: MempoolManager, tx_id: bytes32):
 
 
 @pytest_asyncio.fixture(scope="function")
-async def wallet_node():
+async def wallet_node_sim_and_wallet():
```
[the remark this answers is not in the document]
Comment (aqk) [reply tests/wallet/cat_wallet/test_cat_wallet.py:34]: Ah, yes, you are right. I'll get in another PR.

[the remark this answers is not in the document]
Comment (altendky) [reply tests/wallet/cat_wallet/test_cat_wallet.py:34]: I meant that I looked and it was literally unused.  Did I miss a use case?

File: tests/wallet/test_wallet.py [hunk 2]
```diff
@@ -27,6 +27,12 @@ async def wallet_node():
         yield _
```
Comment (altendky) [inline tests/wallet/test_wallet.py:27]: Looks like this is now unused as well?

**Review by altendky — APPROVED** [review]

**Pull request merged**